### PR TITLE
[15.0][MIG] project_timesheet_time_control : Migration to 15.0

### PR DESCRIPTION
**Pull request by manuelcalerosolis**
This is the migration of project_timesheet_time_control to 15.0

This module has a dependency on : hr_timesheet_task_domain (https://github.com/OCA/timesheet/pull/499)

### CODE REVIEW

**Comment by rousseldenis** [comment]
@manuelcalerosolis Could you check tests?

If this depends on an umerged module, don't hesitate to mention it in this PR first message

**Comment by manuelcalerosolis** [comment]
> @manuelcalerosolis Could you check tests?
> 
> If this depends on an umerged module, don't hesitate to mention it in this PR first message

Done!

**Comment by pedrobaeza** [comment]
This can be moved to OCA/timesheet.

File: project_timesheet_time_control/README.rst [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+
+Note: All the *Start/Resume/Stop* features are disabled if you don't belong to
+the *Timesheets/User* group or if you are viewing a timesheet that belongs
+to another user.
+
+Known issues / Roadmap
+======================
+
+* Rename to ``hr_timesheet_time_control``.
+* Move to `OCA/timesheet <https://github.com/OCA/timesheet>`__.
```
Comment (gurneyalex) [inline project_timesheet_time_control/README.rst:108]: could these two items be tackled as part of the migration?

**Review by gurneyalex — APPROVED** [review]

**Comment by gurneyalex** [comment]
The dependency PR was merged, can you rebase this one to get a proper CI run?

**Comment by ikapasi-initos** [comment]
@manuelcalerosolis The dependent module 'hr_timesheet_task_domain' is merged into the main branch. Rebase this branch with the main branch to resolve the checks issue.

**Comment by ikapasi-initos** [comment]
@manuelcalerosolis can you rebase this pr?

**Comment by pedrobaeza** [comment]
Superseded by #989

**Pull request closed without merging**